script/benchmarks/send_assessments_job_benchmark.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# https://ruby-doc.org/stdlib-2.5.0/libdoc/benchmark/rdoc/Benchmark.html
+# https://github.com/evanphx/benchmark-ips
+# https://github.com/SamSaffron/memory_profiler
+# https://github.com/tmm1/stackprof
+
+require_relative '../../config/environment'
+require 'memory_profiler'
+require 'benchmark/ips'
+
+ActionMailer::Base.perform_deliveries = false
+
+timestamp = Time.now.utc.iso8601
+stackprof_file = "script/benchmarks/output/send_assessments_job_benchmark_#{timestamp}_CPU.dump"
+flamegraph_file = "script/benchmarks/output/send_assessments_job_benchmark_#{timestamp}_FLM"
+memprof_file = "script/benchmarks/output/send_assessments_job_benchmark_#{timestamp}_MEM.log"
+benchmark_file = "script/benchmarks/output/send_assessments_job_benchmark_#{timestamp}_BCM.log"
+$stdout = File.new(benchmark_file, 'w')
+$stdout.sync = true
+
+Benchmark.bm(20) do |x|
```

Comment:
I found that at least on my local machine, changing this to `Benchmark.ips` breaks the stackprof output completely. It would be useful if someone could verify this. If we decide to prefer IPS over stackprof then I can change this test.